src/types/metaFactory.ts

```diff
@@ -117,5 +130,39 @@ export const MetaFactoryTypes: MetaFactoryTypes = {
           }
         }
       })}
+  },
+  candyMachine: {
+    creationMetadataToNftFactory: ({ metadataAccount, metaFactory }: { metadataAccount: Metadata, metaFactory: MetaFactory }, autoApprove: boolean) => {
+      return {
+        id: metadataAccount.mintAddress.toBase58(),
+        contractType: NFTContractTypeName.candyMachine,
+        platformId: metaFactory.platformId,
+        standard: NFTStandard.METAPLEX,
+        name: metadataAccount.name,
+        symbol: metadataAccount.symbol,
+        autoApprove, 
+        approved: autoApprove, 
+        typeMetadata: {
+          ...metaFactory.typeMetadata,
+          overrides: {
+            ...metaFactory.typeMetadata?.overrides,
+            extractor: {
+              id: {
+                extractor: IdExtractorTypes.USE_METAFACTORY_AND_TITLE_EXTRACTOR,
+                params: { 
+                  metaFactory: metaFactory.id,
+                  titleExtractor: metaFactory.typeMetadata?.overrides.extractor?.title
+                }
+              },
+              ...metaFactory.typeMetadata?.overrides.extractor,
```

Comment:
So, the first specified `titleExtractor` sets a titleExtractor field here in the params for this parameterized extractor from the metaFactory

The second `...metaFactory.typeMetadata?.overrides.extractor` sets the usual titleExtractor override in this nftFactory also coming from the metaFactory.

The first one look unused (the implementation of the USE_METAFACTORY_AND_TITLE_EXTRACTOR is pulling the titleStrategy from the second set field, not the first)

So, todos:
 - Decide which one we think is better? I think the second one which is actually being used is maybe better?
 - Remove the first one, since it's unused?